src/io/flutter/preview/PreviewView.java

```diff
@@ -384,20 +400,33 @@ public void customizeCellRenderer(
 
     // Render the variable.
     if (outline.getVariableName() != null) {
+      append(" ");
       appendSearch(outline.getVariableName(), SimpleTextAttributes.REGULAR_ATTRIBUTES);
     }
 
     // Append all attributes.
     final List<FlutterOutlineAttribute> attributes = outline.getAttributes();
     if (attributes != null) {
-      for (FlutterOutlineAttribute attribute : attributes) {
+      if (attributes.size() == 1) {
+        final FlutterOutlineAttribute attribute = attributes.get(0);
         append(" ");
-        append("[", SimpleTextAttributes.GRAYED_ATTRIBUTES);
-        appendSearch(attribute.getName(), SimpleTextAttributes.GRAYED_ATTRIBUTES);
-        append(": ", SimpleTextAttributes.GRAYED_ATTRIBUTES);
         appendSearch(attribute.getLabel(), SimpleTextAttributes.GRAYED_ATTRIBUTES);
```

Comment:
It looks good for `Text` and `Tab`.
But there might be cases when it is not obvious which attribute is set, e.g. `appBar: AppBar false`. What is `false`?

Reply:
I agree that it doesn't work well with just `bool`s. I think it's terser and still understandable with things like `CrossAxisAlignment.stretch`.

Let me try just eliding `data: ` and see how that looks.

Reply:
I simplified this to always eliding `data:`, and only eliding `text:` when it's the only attribute.